examples/gno.land/p/gnoland/commondao/record.gno

```diff
@@ -0,0 +1,178 @@
+package commondao
+
+import (
+	"errors"
+	"math"
+	"std"
+
+	"gno.land/p/demo/avl"
+)
+
+// ErrVoteExists indicates that a user already voted.
+var ErrVoteExists = errors.New("user already voted")
+
+type (
+	// Vote defines a single vote.
+	Vote struct {
+		Address std.Address
+		Choice  VoteChoice
+	}
```

Comment:
What if we want to add more information to the vote? example: A DAO implementation that needs to specify a reasoning about your vote.

Reply:
Good point, I will take a look at this, it would be nice to have more flexibility

Reply:
Addressed in c327ced and 88a0195